tests/unittest/_torch/modeling/test_modeling_nemotron_nano_v2_vl.py

```
@@ -260,3 +263,202 @@ def test_encode_multimodal_unknown_modality_raises(self):
+            self._make_video_param(has_audio=True),
+        ]
+
+        # Build a minimal mock of NemotronH_Nano_VL_V2 with only the
+        # attributes _encode_multimodal touches.
+        model = MagicMock()
+        model.vision_encoder = MagicMock(side_effect=lambda p: next(vision_returns))
+        model.sound_encoder = MagicMock()  # not None → audio path taken
+        model._encode_audio_data = MagicMock(side_effect=lambda d: next(audio_returns))
+
+        # Call the real method, bound to our mock
+        embeddings, num_tokens = NemotronH_Nano_VL_V2._encode_multimodal(model, params)
```

Ruff (0.15.9)

[warning] Unpacked variable `num_tokens` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)